demo/src/app/components/+pagination/demos/content-switching/content-switching.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'demo-pagination-content-switching',
+  templateUrl: './content-switching.html'
+})
+export class DemoPaginationContentSwitchingComponent implements OnInit {
+  contentArray = Array.apply(null, Array(88)).map((v: any, i: number) => `Content line ${i + 1}`);
+  returnedArray: string[];
+
+  ngOnInit(): void {
+    if (!this.returnedArray){
```

Comment:
why do you need this check if `returnedArray` is undefined by default?

Reply:
@IlyaSurmay just habit